src/routes/categories/[topicSlug]/[tableSlug]/[categorySlug].svelte

```diff
@@ -0,0 +1,153 @@
+<script>
+  import BasePage from "../../../../ui/BasePage.svelte";
+
+  import Header from "../../../../ui/Header.svelte";
+  import Map from "../../../../ui/map/Map.svelte";
+  import InteractiveLayer from "../../../../ui/map/InteractiveLayer.svelte";
+  import TileSet from "../../../../ui/map/TileSet.svelte";
+  import BoundaryLayer from "../../../../ui/map/BoundaryLayer.svelte";
+  import DataLayer from "../../../../ui/map/DataLayer.svelte";
+  import config from "../../../../config";
+  import TopicExplorer from "../../../../ui/TopicExplorer.svelte";
+  import Topic from "../../../../ui/Topic.svelte";
+  import Feedback from "../../../../ui/Feedback.svelte";
+  import {
+    getLadName,
+    updateSelectedGeography,
+    updateHoveredGeography,
+    selectedGeography,
+  } from "../../../../model/geography/geography";
+  import { categoryDataIsLoaded, categoryData } from "../../../../model/censusdata/censusdata";
+  import { appIsInitialised } from "../../../../model/appstate";
+
+  import { page } from "$app/stores";
+  import { goto } from "$app/navigation";
+
+  let { topicSlug, tableSlug } = $page.params;
+
+  let locationId = $page.query.get("location");
+  let locationName;
+
+  $: {
+    if ($selectedGeography.lad) {
+      $page.query.set("location", $selectedGeography.lad);
+      goto(`?${$page.query.toString()}`);
+      locationId = $page.query.get("location");
+      locationName = getLadName(locationId);
+    }
+  }
+
+  $: appIsInitialised, $appIsInitialised && initialisePage();
+
+  function initialisePage() {
+    if (locationId) {
+      locationName = getLadName(locationId);
+      updateSelectedGeography(locationId);
+    }
+  }
+
+  $: innerWidth = 0;
+</script>
+
+<svelte:window bind:innerWidth />
+<svelte:head>
+  <title>2021 Census Data Atlas Categories</title>
+</svelte:head>
+
+<BasePage mobileMap={false} withoutBackground>
+  <span slot="header">
+    <Header
+      showBackLink
+      serviceTitle="Choose a data option {locationId ? `for ${locationName}` : ''}"
+      description="Choose a category and select an option within it to explore {locationName
+        ? `${locationName}'s`
+        : 'Census'} data."
+    />
+  </span>
+
+  <span slot="map">
```

Comment:
I don't think its in-scope for this PR, but this map element looks very similar to that used in other screens, could this be made importable from a common file, somehow?

Reply:
This is a really good point, we could have the map element as its own svelte component and best to have a separate ticket for this as well. 👍